src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -208,6 +208,10 @@ def test_WINNF_FT_S_SIQ_3(self):
         'lowFrequency': 3650000000,
```

Comment:
The changes in this file all seem logically correct, assuming that the new function is correctly implemented.

Reply:
Thanks.